build-configs.yaml

```diff
@@ -423,7 +423,7 @@ build_configs:
 
   android_4.14-r:
```

Comment:
Please update the config name accordingly without `-r` to follow the convention.

Reply:
ok, done.

Reply:
thanks